src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.Test;
+
+
+public class UtilsTest {
+
+    @Test
+    public void nullObject_returnsTrue() {
+        final Object[] inputs = { "", null, "\n  \n" };
+        final boolean resultMessage = true;
+        assertEquals(resultMessage, Utils.isAnyNull(inputs));
+    }
+    
+    @Test
+    public void noNullObject_returnsFalse() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.